conformance/utils/kubernetes/helpers.go

```diff
@@ -684,14 +749,15 @@ func parentsForRouteMatch(t *testing.T, routeName types.NamespacedName, expected
 			return false
 		}
 		if !reflect.DeepEqual(aParent.ParentRef.Group, eParent.ParentRef.Group) {
-			tlog.Logf(t, "Route %s expected ParentReference.Group to be %v, got %v", routeName, eParent.ParentRef.Group, aParent.ParentRef.Group)
+			tlog.Logf(t, "Route %s expected ParentReference.Group to be %v, got %v", routeName, *eParent.ParentRef.Group, *aParent.ParentRef.Group)
```

Comment:
If Group (and Kind below) are by some reason null, the whole test will panic (I have tested below).  You may want to do something like:

`tlog.Errorf(t, "something happened in %v", ptr.Deref(Xpto, gatewayv1.Kind("")))`


```
var (
	Xpto *gatewayv1.Kind
)

func TestBla(t *testing.T) {
	//Xpto = ptr.To(gatewayv1.Kind("xxxx"))
	tlog.Errorf(t, "something happened in %v", *Xpto)
}
```